test/functional/testUtil/diffPrinter.js

```diff
@@ -0,0 +1,111 @@
+class DiffPrinter {
+  /* eslint-disable lodash/prefer-constant */
+  static get ANSI_RED() { return '\u001B[31m'; }
+  static get ANSI_GREEN() { return '\u001B[32m'; }
+  static get ANSI_GREY() { return '\u001B[90m'; }
+  static get ANSI_RESET() { return '\u001B[0m'; }
+  /* eslint-enable lodash/prefer-constant */
+
+  /**
+   * Prints line of text in colour provided
+   * @param {string} text text to print, default: no text
+   * @param {string} colour colour of text, default: no colour
+   */
+  static printLine(text = '', colour = 'none') {
+    let ansiEscCode = '';
+    switch (colour) {
```

Comment:
Callers could probably just pass in `ANSI_RED`, `ANSI_GREEN`, etc directly if they are constants.

Reply:
I'm thinking that the callers should not know that we're using ANSI escape code to render colour.

Reply:
You have a point.